blocks/header/header.css

```diff
@@ -239,7 +238,6 @@ header nav .nav-sections a {
   header nav .nav-sections a {
     font-size: var(--body-font-size-s);
```

Comment:
@sforschen, I'm wondering if using a larger font size for desktop might work well here  
--body-font-size-m 
and removing the font-weight
font-weight: unset;